src/pages/Intents/IntentIdentifiers.js

```diff
@@ -81,38 +76,36 @@ const IntentIdentifiers = ({
         <div className={styles.intentFormGrid}>
             <TextField
                 required
-                name="shortName"
-                label={i18n.t('Short name')}
+                name="name"
+                label={i18n.t('Intent name')}
                 onChange={(e) => onChange('name', e.value)}
                 value={formData.name || ''}
             />
 
             <TextField
                 name="description"
-                label={i18n.t('Description')}
+                label={i18n.t('Intent description')}
                 onChange={(e) => onChange('description', e.value)}
                 value={formData.description || ''}
             />
 
-            {!edit && (
```

Comment:
I didn't think this should be conditionally hidden on edit, but maybe I missed something when looking at the specs?